pipes/WDL/workflows/merge_vcfs.wdl

```diff
@@ -0,0 +1,11 @@
+version 1.0
+
+import "../tasks/tasks_interhost.wdl" as interhost
+
+workflow mafft {
```

Comment:
workflow name doesn't match filename

Reply:
Separately, should we provide the option to run snpEff after merge?

Reply:
Ah, yes. I'll add this.